apps/coscrad-frontend/src/components/resources/songs/song-detail.thumbnail.presenter.tsx

```diff
@@ -20,7 +20,10 @@ export const SongDetailThumbnailPresenter = ({
     <div data-testid={title}>
         <Card className="detail-card">
             <div id="detail-term" className="detail-meta">
-                {formatBilingualText(title, titleEnglish)}
+                <BilingualTextPresenter
```

Comment:
One question I have had looking at the front-end is whether it should be the `BilingualTextPresenter`'s responsibility to take in multilingual text and apply the default language code and search pattern.